src/lib/sfge/AbstractSfgeEngine.ts

```diff
@@ -91,17 +124,17 @@ export class SfgeEngine extends AbstractRuleEngine {
 
 		partialRules.forEach(({name, description, category}) => {
 			completeRules.push({
-				engine: ENGINE.SFGE,
+				engine: AbstractSfgeEngine.ENGINE_ENUM,
 				sourcepackage: "sfge",
 				name,
 				description,
-				// SFGE rules each belong to exactly one category, so the string must be converted to a singleton array.
+				// Graph Engine rules each belong to exactly one category, so the string must be converted to a singleton array.
```

Comment:
Is this still true? Especially for the AbstractSfgeEngine?

Reply:
Still true. Each rule has exactly one category, but the rule could also be DFA or non-DFA.